sphinx_gallery/gen_gallery.py

```diff
@@ -143,28 +161,55 @@ def touch_empty_backreferences(app, what, name, obj, options, lines):
         open(examples_path, 'w').close()
 
 
-gallery_conf = {
-    'filename_pattern': re.escape(os.sep) + 'plot',
-    'examples_dirs': '../examples',
-    'gallery_dirs': 'auto_examples',
-    'mod_example_dir': os.path.join('modules', 'generated'),
-    'doc_module': (),
-    'reference_url': {},
-}
+def sumarize_failing_examples(app, exception):
+    """Embed hyperlinks to documentation into example code"""
+    if exception is not None:
+        return
+
+    gallery_conf = app.config.sphinx_gallery_conf
+    failing_examples = set(gallery_conf['failing_examples'])
+    ex_fail_examples = set(gallery_conf['expected_failing_examples'])
+
+    still_ex_fails = failing_examples.intersection(ex_fail_examples)
+    expected_fail_msg = []
+    if still_ex_fails:
+        expected_fail_msg.append("Known failing examples, still failing:")
+        for fail_example in still_ex_fails:
+            expected_fail_msg.append(fail_example + ' failed leaving traceback:\n' +
+                                     gallery_conf['failing_examples'][fail_example] + '\n')
+        print("\n".join(expected_fail_msg))
+
+    unexpected_ex_fails = failing_examples.difference(ex_fail_examples)
+    fail_msgs = []
+    if unexpected_ex_fails:
+        fail_msgs.append("Unexpected failing examples:")
+        for fail_example in unexpected_ex_fails:
+            fail_msgs.append(fail_example + ' failed leaving traceback:\n' +
+                             gallery_conf['failing_examples'][fail_example] + '\n')
+
+    unexpected_ex_pass = ex_fail_examples.difference(failing_examples)
+    if unexpected_ex_pass:
+        fail_msgs.append("Examples that are not failing anymore:\n" +
```

Comment:
"Examples that ran without error but were expected to fail:\n"

Maybe we can add some instruction to give a hint about what to do in conf.py, e.g. say that the examples in `unexpected_ex_pass` can be removed from the dictionary in conf.py.